MetaMask/metamask-extension
# MV3 artifacts e2e measurement times

## Explanation
We need a way to measure the delay caused by a service-worker re-start on common user actions along the application.
For that, this PR will take care of measuring common action times, on an MV2 test build, so we have this time reference. Actions covered:

- how much time it takes for loading a newly created account (from Create Account button until balance is loaded
- how much time it takes from clicking Confirm tx until Tx is confirmed

**Approach**: this task will be done in 2 PRs:
- The 1st PR (#15283) created the e2e test for performing the actions of Load an Account and Send a Tx.
- The 2nd PR (this one) will create a build artifact for exporting the measurements into a json file.

## More Information
* Closes #15292

## Manual Testing Steps
Check the build artifacts created on this branch by Metamaskbot, and ensure that there is a file mv3 that captures mv3 e2e actions stats.

## Pre-Merge Checklist


### + If there are functional changes:
